lib/tasks/demo_data_for_development.rake

```diff
@@ -0,0 +1,70 @@
+namespace :data do
+  desc 'Create demo data for our local development'
+  task test: :environment do
+    include FactoryGirl::Syntax::Methods
+
+    def generate_events program, conference
+        user1 = create(:user)
+        user2 = create(:user)
+
+        selected_schedule = create(:schedule, program: program)
+        demo_schedule = create(:schedule, program: program)
+        program.update_attributes(selected_schedule: selected_schedule)
+
+        build(:event, program: program, title: 'Demo Event', abstract: 'This is a demo event instance whose state not defined.').save(validate: false)
```

Comment:
No validation fails when event is scheduled,it fails when event is created because we are creating events of conference that has already happened.
Actually let's say we create a conference that has happened in the past.Our conference is like this:
> {start_date:22,end_date:24}

And now when we create its events following validation won't pass:

`errors.
        add(:created_at, "can't be after the conference end date!") if program.conference && program.conference.end_date &&
        (**Date.today** > program.conference.end_date)`

because this validation checks the current date(Date.today) and compares it with conference end_date.